pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2316,6 +2316,7 @@ type SSSDSidecarAdditionalFile struct {
 
 // NetworkSpec for Ceph includes backward compatibility code
 // +kubebuilder:validation:XValidation:message="at least one network selector must be specified when using multus",rule="!has(self.provider) || (self.provider != 'multus' || (self.provider == 'multus' && size(self.selectors) > 0))"
+// +kubebuilder:validation:XValidation:message="the legacy hostNetwork settingcan only be set if the network.provider is set to the empty string",rule="!has(self.hostNetwork) || !has(self.provider)"
```

Comment:
nit: typo "settingcan"

Reply:
@travisn  wrote:
> nit: typo "settingcan"

I had already spotted and fixed this.